### ISSUE TITLE
README: Not clear why serial port should be 'enabled'

### ISSUE THREAD

**Issue by ybakos** [osu-cascades/dio-controllers#17]
The README contains instructions to "enable the serial port" but this is unclear, does not make sense, and could be the source of the recent I/O issue.

@MakeilaLundy @JWTappert Can you please explain why the instructions indicate that the user should enable ssh login over the serial port? Because that is the setting that the README is alluding to, albeit with the misleading title of "enable serial port."

In other words, why? What for? Shouldn't it be *disabled*?

The last line "The serial port is now ready to be hooked up!" makes no sense. Hooked up to what? What for? Why? (Again, shouldn't ssh over serial be _disabled_?)
